SOURCES/sympa-httpd-fcgid-el7.conf

```diff
@@ -2,10 +2,10 @@
 
 ## Definition of Sympa FastCGI server.
 <IfModule mod_fcgid.c>
-    IPCCommTimeout 300
+    FcgidIOTimeout 300
     # You wish to increase the value in following line, don't you?
-    MaxProcessCount 5
-    MaxRequestLen 131072
+    FcgidMaxProcesses 5
+    FcgidMaxRequestLen 131072
```

Comment:
See a comment on #9.

Reply:
The rationale for keeping FcgidIOTimeout and FcgidMaxRequestLen but discard FcgidMaxProcesses is the formers are valid inside a VirtualHost while the later is not.
I'll add another patch to plain remove them and add a comment.